src/components/FilePicker/FilePicker.tsx

```diff
@@ -35,6 +35,10 @@ interface FilePickerProps extends MarginProps, ComponentPropsWithoutRef<'input'>
      * it works better when short
      */
     buttonText: string;
+    /**
+     * Keep the action button displayed instead of showing the success state.
+     */
+    keepActionButton?: boolean;
```

Comment:
Alternative naming to consider `alwaysShowBrowseButton`? 

For me, it sounds better because we do not assume our user knows that we hide the button after the selection

Reply:
`alwaysShowActionButton` sounds right